dbt/include/athena/macros/materializations/models/table/table_hive_ha.sql

```diff
@@ -38,9 +38,7 @@
     {% set tmp_relation = make_temp_relation(target_relation, '__ha') %}
 
     -- drop the tmp_relation
-    {% call statement('drop_tmp_relation', auto_begin=False) -%}
-      drop table if exists {{ tmp_relation.render_hive() }}
-    {%- endcall %}
+    {{ drop_relation(tmp_relation) }}
```

Comment:
Could you please replace this drop with a call to delete_from_glue_catalog? As you can see this is just a normal drop, drop_relation macro also takes care of deleting s3.. Not really sure why I implemented like that, but there could be edge cases where we don't want to have data in s3 remove.
Therefore this implementation is different from the original one

Reply:
Sorry, you've mentioned this a few times and I didn't recognize it. I've changed it. It also seems like the following line can use this delete since it was using the same drop statement, but let me know if you think otherwise:
https://github.com/dbt-athena/dbt-athena/blob/f08552c016a4718897a1c7807ae1a212c4d79ed2/dbt/include/athena/macros/materializations/models/table/table_hive_ha.sql#L54-L55

Reply:
Not sure what you mean. Isn't the full refresh cleanup happening here:
https://github.com/dbt-athena/dbt-athena/blob/f08552c016a4718897a1c7807ae1a212c4d79ed2/dbt/include/athena/macros/materializations/models/table/table_hive_ha.sql#L29-L31

Reply:
I see, yeah that's how it's currently implemented.